app/controllers/events_controller.rb

```diff
@@ -1,9 +1,9 @@
 class EventsController < ApplicationController
   def index
     if params[:websource].present?
-      @events = Event.where("websource LIKE ?", params[:websource]).paginate(:page => params[:page], :per_page => 10)
+      @events = Event.where("websource LIKE ?", params[:websource])
     else
-      @events = Event.all.paginate(:page => params[:page], :per_page => 10)
+      @events = Event.all
```

Comment:
@moizansari22 Why you have removed pagination?

Reply:
added will_paginate functionality and written its Rspec script as per requirments